guided-match/ddl.sql

```diff
@@ -0,0 +1,207 @@
+  journey_id                        UUID PRIMARY KEY,
+  journey_name                      VARCHAR(200),
+  published_status                  CHAR(1), -- Y(es) or N(o)
+  parent_journey_id                 INTEGER
+);
+CREATE INDEX JOUR_IDX1 on JOURNEYS(journey_name);
+CREATE INDEX JOUR_IDX2 on JOURNEYS(parent_journey_id);
+
+/* Neo4J
+CREATE TABLE lot_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   lot_number                       VARCHAR(10) NOT NULL        
+);
+*/
+
+
+
+/* Neo4J
+CREATE TABLE question_definitions (
+  question_id                       INTEGER PRIMARY KEY,
+  question_type                     VARCHAR(20) NOT NULL,
+  question_description              VARCHAR(2000) NOT NULL,
+  question_text                     VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL, -- Y(es) or N(o),
+  validate_via_agreement            CHAR(1) NOT NULL, -- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_groups (
+  question_group_id                 INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(100) NOT NULL,
+  published_status                  CHAR(1) NOT NULL -- Need to check if we don't want to make this boolean
+);
+*/
+/* Neo4J
+CREATE TABLE question_group_members (
+  question_group_member_id          INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL-- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_outcome_evaluators (
+  outcome_evaluator_code            VARCHAR(10) PRIMARY KEY,
+  outcome_evaluator_description     VARCHAR(200) NOT NULL
+);
+*/
+
+CREATE TABLE search_domains (
+  domain_modifier_id               INTEGER PRIMARY KEY,
+  search_id                        INTEGER NOT NULL,
+  journey_id                       UUID NOT NULL,
+  modifier_journey_name            VARCHAR(20),
+  modifier_journey_description     VARCHAR(2000)
+);
+
+create index SEDO_IDX1 ON SEARCH_DOMAINS(search_id);
+create index SEDO_IDX2 ON SEARCH_DOMAINS(journey_id);
+create index SEDO_IDX3 ON SEARCH_DOMAINS(modifier_journey_name);
+
+CREATE TABLE search_terms (
+  search_id                        INTEGER PRIMARY KEY,
+  search_term                      VARCHAR(50) NOT NULL
+);
+
+CREATE INDEX SETE_IDX1 ON SEARCH_TERMS (search_term);
+
+/* Neo4J
+CREATE TABLE standard_answers  (
+  standard_answer_id                INTEGER PRIMARY KEY,
+  valid_value                       VARCHAR(200),
+  answer_label                      VARCHAR(100),
+);
+*/
+
+/* Referential constraints */
+
+ALTER TABLE journey_instance_questions 
+ADD CONSTRAINT journey_instance_questions_journey_instances_fk FOREIGN KEY (journey_instance_id) 
+    REFERENCES journey_instances (journey_instance_id); 
+    
+ALTER TABLE journey_instance_questions 

```

Comment:
`journey_question_id` is a UUID from the graph - no FK required here

Reply:
Removed fk refrence that wasn't needed.

Reply:
I think this should be:
```
ALTER TABLE journey_instance_questions 
ADD CONSTRAINT journey_instance_questions_journey_instance_fk FOREIGN KEY (journey_instance_id) 
    REFERENCES journey_instances (journey_instance_id);
```
?